docs/diagrams/flow-finding-a-job.html

```
@@ -0,0 +1,156 @@
+
+      <!-- 1. U -> H -->
+      <path d="M140,140 H384" fill="none" stroke="#505050" stroke-width="1" marker-end="url(#arrow)"/>
+      <rect x="200" y="122" width="84" height="12" rx="2" fill="#fdfefc"/>
+      <text x="242" y="131" fill="#141414" font-size="8" font-family="'Geist Mono', monospace" text-anchor="middle" letter-spacing="0.06em">GET SEARCH</text>
+
+      <!-- 2. H -> F -->
+      <path d="M384,196 H616" fill="none" stroke="#505050" stroke-width="1" marker-end="url(#arrow)"/>
+      <rect x="458" y="178" width="84" height="12" rx="2" fill="#fdfefc"/>
+      <text x="500" y="187" fill="#141414" font-size="8" font-family="'Geist Mono', monospace" text-anchor="middle" letter-spacing="0.06em">BUILD FILTER</text>
+
+      <!-- 3. F --> H (return, dashed) -->
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)